shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/spi/handler/PipelineMetaDataChangedHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.core.spi.handler;
+
+import org.apache.shardingsphere.elasticjob.infra.pojo.JobConfigurationPOJO;
+import org.apache.shardingsphere.infra.util.spi.annotation.SingletonSPI;
+import org.apache.shardingsphere.mode.repository.cluster.listener.DataChangedEvent;
+
+import java.util.regex.Pattern;
+
+/**
+ * Pipeline meta data changed handler.
+ */
+@SingletonSPI
+public interface PipelineMetaDataChangedHandler {
+    
+    /**
+     * Get key pattern.
+     *
+     * @return key pattern
+     */
+    Pattern getKeyPattern();
+    
+    /**
+     * Event changed handler.
+     *
+     * @param event changed event
+     * @param jobConfigPOJO job config pojo
+     */
+    void handle(DataChangedEvent event, JobConfigurationPOJO jobConfigPOJO);
```

Comment:
Since it's not just used for job, so `JobConfigurationPOJO jobConfigPOJO` param is not suitable to put here